database/migrations/2022_02_01_105334_add_2022_contests.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\Schema;
```

Comment:
@Podly27 Maybe I could/should remove this. It's actually not used. We're not changing schema anywhere in the migration...

Reply:
I removed the unused Schema include, so I am marking this as resolved.